src/components/Pane/WrappedContent.tsx

```diff
@@ -0,0 +1,29 @@
+/* tslint:disable:no-unused-variable */
+import * as React from 'react';
+/* tslint:enable:no-unused-variable */
+
+export interface IWrappedContentProps extends React.Props<WrappedContent> {
+  /**
+   * Optional class name to be added to the root class
+   */
+  className?: string;
+}
+
+export class WrappedContent extends React.Component<IWrappedContentProps, any> {
```

Comment:
i don't understand the point of this class.. it's just a div... why not use a div?